seqr/views/utils/orm_to_json_utils.py

```diff
@@ -757,10 +757,30 @@ def get_json_for_matchmaker_submission(submission):
         additional_model_fields=['contact_name', 'contact_href', 'submission_id'])
 
 
-def get_json_for_rna_seq_outliers(models, **kwargs):
-    additional_values = {
-        'isSignificant': Case(
-            When(p_adjust__lt=models.model.SIGNIFICANCE_THRESHOLD, then=Value(True)), default=Value(False)
-        ),
-    }
-    return get_json_for_queryset(models, additional_values=additional_values, **kwargs)
+def get_json_for_rna_seq_outliers(filters, significant_only=True):
+    filters['sample__is_active'] = True
```

Comment:
its not super important, but there is always a risk associated with modifying the parameters passed to functions - while `filters` is never used again in the calling function, it would be confusing if you did try to reuse it and it was different. Changing this to copy the values is more stable: `filters  = {'sample__is_active': True, **filters}`